core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/annotations/Plugin.kt

```diff
@@ -28,5 +28,18 @@ public annotation class Import
 @Target(AnnotationTarget.PROPERTY)
 public annotation class Order(val order: Int)
 
+/**
+ * For internal use
+ * Compiler plugin materializes schemas as classes.
+ * These classes have two kinds of properties:
+ * 1. Scope properties that only serve as a reference for internal property resolution
+ * 2. Schema properties that reflect dataframe structure
+ * Scope properties need
+ * to be excluded in IDE plugin and in [org.jetbrains.kotlinx.dataframe.codeGen.MarkersExtractor.get]
+ * This annotation serves to distinguish between the two where needed
+ */
+@Target(AnnotationTarget.PROPERTY)
+public annotation class ScopeProperty
```

Comment:
do you use this annotation already somewhere? I'm curious to see how/where it's used :)

Reply:
What do you mean by use? :o 
Here generated property is annotated 
plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/TokenGenerator.kt

Reply:
Plus other things from the same commit

Reply:
Ooh so this annotation class is only used in generated properties. That wasn't clear to me